planning.ipynb

```diff
@@ -342,22 +515,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As we can see, it hasn't completed the goal. Now, we define the sequence of actions that it should take in order to achieve\n",
-    "the goal. Then the `airCargo` acts on each of them."
+    "It will give us False because the goal state is not yet reached. Now, we define the sequence of actions that it should take in order to achieve the goal. Then the `airCargo` acts on each of them.\n",
+    "\n",
+    "Let us have an idea of the actions available to us i.e, Load, Unload, Fly\n",
```

Comment:
Instead of this sentence, write something like: "The actions available to us are the following:"